x-pack/plugins/security_solution/server/endpoint/routes/policy/handlers.ts

```diff
@@ -63,3 +64,19 @@ export const getAgentPolicySummaryHandler = function (
     });
   };
 };
+
+export const getPolicyListHandler = function (
+  endpointAppContext: EndpointAppContext
+): RequestHandler<undefined, GetPackagePoliciesRequest['query'], undefined> {
+  return async (context, request, response) => {
+    const soClient = context.core.savedObjects.client;
+    const packagePolicyService = endpointAppContext.service.getPackagePolicyService();
+    const doc = await packagePolicyService.list(soClient, request.query);
+    if (doc) {
+      return response.ok({
+        body: doc,
+      });
+    }
```

Comment:
this should be wrapped in a `try {} catch (error) {}`  and the catch of the error should then do this:

```suggestion
return response.notFound({ body: new EndpointError('Failed to retrieve package policy list', error) });
```

Which does a few thing:

1. it returns back to the consumer of this API a `body` that is consistent with other errors (normally it reaches the UI code as an object containing the `message` property)
2. it helps the kibana logs to see the original code that was encurred when we called the fleet service